src/steps/mod.rs

```diff
@@ -8,18 +8,26 @@ pub use integrator::IntegratorPlugin;
 pub use prepare::PreparePlugin;
 pub use solver::SolverPlugin;
 
-use bevy::prelude::SystemLabel;
+use bevy::prelude::SystemSet;
 
 /// The main steps in the physics simulation loop.
-#[derive(SystemLabel, Clone, Copy, Debug, PartialEq, Eq, Hash)]
-pub enum PhysicsStep {
+#[derive(SystemSet, Clone, Copy, Debug, PartialEq, Eq, Hash)]
+pub enum PhysicsSet {
     /// In the preparation step, necessary preparations and updates will be run before the rest of the physics simulation loop.
     /// For example, bevy `Transform`s are synchronized with the physics world, AABBs are updated etc.
     Prepare,
     /// During the broad phase, potential collisions will be collected into the [`BroadCollisions`] resource using simple AABB intersection checks. These will be further checked for collision in the [`PhysicsStep::NarrowPhase`].
     ///
     /// The broad phase speeds up collision detection, as the number of accurate collision checks is greatly reduced.
     BroadPhase,
+    /// Substepping is an inner loop inside a physics step, see [`PhysicsSubstep`] and [`XpbdSubstepSchedule`]
+    Substeps,
+    // todo: what about sync transforms?
```

Comment:
this seems to happen some way, though I don't know how... removing todo.

Reply:
This is being done in the `Prepare` step, at the start of the simulation loop, as explained in the doc comment. I think originally it was being done at the end of the loop, but I think it can be done at the start as well?

Reply:
Doing it at the end would improve latency, but let's revisit it in another PR.